examples/next-pages-apollo-authentication/src/pages/index.js

```diff
@@ -0,0 +1,136 @@
+import Link from "next/link";
+import { useAuth } from "@/lib/auth/AuthProvider";
+import { gql, useQuery } from "@apollo/client";
+
+const GET_USER_SETTINGS = gql`
+  query GetUserSettings {
+    viewer {
+      email
+      username
+    }
+  }
+`;
+
+export default function Home() {
+  const { tokens, isLoading, logout, refreshAuth } = useAuth();
+  const isAuthenticated = !!tokens?.authToken;
+
+  const { data, loading: userDataLoading } = useQuery(GET_USER_SETTINGS, {
+    skip: !isAuthenticated,
+  });
+
+  return (
+    <div className="min-h-screen p-8">
+      <header className="flex justify-between items-center mb-12">
+        <h1 className="text-2xl font-bold">My WordPress App</h1>
+        <div>
+          {isLoading ? (
+            <p>Loading...</p>
+          ) : isAuthenticated ? (
+            <Link
+              href="/dashboard"
+              className="bg-blue-600 text-white px-4 py-2 rounded-md"
+            >
+              Dashboard
+            </Link>
```

Comment:
```suggestion
            <button onClick={logout} className='bg-blue-600 text-white px-4 py-2 rounded-md leading-[normal]'>
              Sign out
            </button>
```
We may replace Dashboard with Sign out button as it don't have any function